adapters/amqp-vertx/pom.xml

```diff
@@ -91,5 +97,25 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>docker-push-image</id>
+      <build>
+          <plugins>
+              <plugin>
+                  <groupId>io.fabric8</groupId>
+                  <artifactId>docker-maven-plugin</artifactId>
+                  <executions>
+                      <execution>
+                          <id>docker-push-image</id>
+                          <phase>install</phase>
```

Comment:
by default, the `push` goal is bould to the `deploy` phase. I assume that you want it in the `install` phase because you usually do not deploy any Hono artifacts into a local repository, right?

Reply:
I've tested this locally and got an error on the integration tests when using the deploy phase. I will check again tomorrow.

Reply:
I'm getting folllowing error when running the deploy lifecycle:
Failed to execute goal org.sonatype.plugins:nexus-staging-maven-plugin:1.6.8:deploy (injected-nexus-deploy) on project hono-tests: The packaging for this project did not assign a file to the build artifact -> [Help 1]

Reply:
When you rebase your PR and use `-DdisableNexus`, then you should be able to revert back to standard maven deploy.